app/views/layouts/rails_admin/application.html.erb

```diff
@@ -8,16 +8,16 @@
     <div class="label label-warning" id="loading" style="display:none; position:fixed; right:20px; bottom:20px; z-index:100000">
       <%= t('admin.loading') %>
     </div>
-    <nav class="navbar navbar-default navbar-fixed-top">
+    <nav class="navbar navbar-expand-xl fixed-top navbar-dark bg-primary border-bottom">
```

Comment:
You're right about `navbar-expand-md`, but `navbar-dark bg-primary` are intentional. That was to make theming functionality(explained later) to stand out when using other themes, instead of just showing grayish navbar.